sdk/storage/azblob/client_test.go

```diff
@@ -193,7 +193,7 @@ func performUploadAndDownloadFileTest(t *testing.T, _require *require.Assertions
 	_, err = client.UploadFile(context.Background(), containerName, blobName, file,
 		&blockblob.UploadFileOptions{
 			BlockSize:   int64(blockSize),
-			Parallelism: uint16(parallelism),
+			Concurrency: uint16(parallelism),
```

Comment:
Seems odd to publicly switch the terminology to Concurrency but internally keep it as parallelism.